x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_list.tsx

```diff
@@ -101,8 +101,11 @@ export const DataFrameAnalyticsList: FC<Props> = ({ isManagementTable }) => {
     isLoading: setIsLoading,
     onRefresh: () => getAnalytics(true),
   });
-  // Call useRefreshInterval() after the subscription above is set up.
-  useRefreshInterval(setBlockRefresh);
+
+  if (isManagementTable === false) {
```

Comment:
Hooks need to be at the top level, we cannot put them inside conditions unfortunately https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level.

Maybe an alternative is to pass `isManagementTable` on to the hook and act on it there?

Reply:
That makes sense normally, but in this case `isManagementTable` value is never changing so it wouldn't change the order of the hooks being called. 🤔 
I'm happy to change it though since maybe it does set an example for a pattern we wouldn't want to follow most of the time.